.gitignore

```diff
@@ -17,7 +17,7 @@ gulp-tsc-tmp-*
 *.suo
 *.user
 *.sln.docstates
-/.vs
+.vs/
```

Comment:
Should it be both the patterns (unless the old one never came) since this is a repo-wide file